packages/admin-client/src/components/mappings/CreateMappingForm.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { 
+  Box, 
+  TextField, 
+  Button, 
+  Dialog, 
+  DialogTitle, 
+  DialogContent, 
+  DialogActions,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+} from '@mui/material';
+import { LiteLLMMapping } from '../../types/litellm-mapping';
+import { LangFlowMapping } from '../../types/langflow-mapping';
+import { createLangFlowMapping, createNewModelLiteLLM } from '../../services/endpoints';
+
+type CreateMappingFormProps = {
+  open: boolean;
+  onClose: () => void;
+};
+
+const CreateMappingForm = ({ open, onClose }: CreateMappingFormProps) => {
+  const [formData, setFormData] = useState<LiteLLMMapping>({
+    provider: 'langflow',
+    url: '',
+    modelName: '',
+    historyComponentID: '',
+    apiKey: '',
+  });
+
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const langflowData: LangFlowMapping= {
+        model: formData.modelName,
+        url: formData.url,
+        historyComponentID: formData.historyComponentID
+      };
+  
+      const langFlowResponse = await createLangFlowMapping(langflowData);
+      const liteLLMResponse = await createNewModelLiteLLM(formData);
```

Comment:
I think we should have the LiteLLM model be added automatically when we create the LangFlow Mapping. ie) not requiring an additional backend call from the frontend, but having the call itself in the backend.